packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -76,10 +77,15 @@ def test_if_uiconfig_is_available(cluster):
 
 
 def test_if_dcos_history_service_is_up(cluster):
-    r = cluster.get('/dcos-history-service/ping')
 
-    assert r.status_code == 200
-    assert 'pong' == r.text
+    @retry(stop_max_delay=20000, wait_fixed=500)
```

Comment:
@cmaloney I don't follow, this is test is sending the exact same GET request to the same adminrouter endpoint as wait_for_dcos_history_up. The only difference is this asserts /ping is working perfectly whereas the wait_for_up check just makes sure its _probably_ somewhere. In any case, this test is hitting /ping so it will have the exact same behavior regardless if it has data